### ISSUE TITLE
Evaluation of governance groups

### ISSUE THREAD

**Issue by sjurl** [ices-eg/DIG#313]

**Comment by sjurl** [comment]
For september interim meeting 2022

**Comment by jensr** [comment]
Apologies for slow reply on this one. 
Overall, I think it would be good to align/identify Governance groups with existing data workflows in ICES (e.g. the 32 diagrams as they emerge), and outline where it is a formal/recognised. @neil-ices-dk did at one point make a list of the number of governance groups and de-facto groups that look after certain workflows. It might make sense to start with that and update it accordingly (e.g. list workflows, responsible group, and perhaps a few other columns like links to their GitHub, whether they have done a governance evaluation of their workflow)
That will provide us with a better oversight in the long term.

**Comment by neil-ices-dk** [comment]
yes - the old list is available as an Excel, have uploaded on [DIG](https://community.ices.dk/Committees/DIG/Master%20documents/Governance%20Evaluations/governance-overview%20(1).xlsx)

![image](https://user-images.githubusercontent.com/23527005/194328196-3bfc398d-5ccc-4fed-bd9a-8dcd71239489.png)

**Comment by sjurl** [comment]
Update overview of governance groups - new groups have been created.

Future plans: make available data governance data flow in a web focused way

**Comment by pcrjoana** [comment]
[governance-overview (1).xlsx](https://github.com/ices-eg/DIG/files/14536307/governance-overview.1.xlsx)

I added some information to the file @neil-ices-dk shared above, including steering groups, and updated some governance groups/added data flows.

@neil-ices-dk are there steering groups for external WG's, like HELCOM and OSPAR groups? Also, who governs ASD? I was thinking ACOM, but I'm not sure. Who would be the SG for ACOM governed data flows, would there even be one?

**Comment by neil-ices-dk** [comment]
@pcrjoana thank you - i have moved the file to the DIG master folders

https://community.ices.dk/Committees/DIG/Master%20documents/Governance%20Evaluations/governance-overview.2024-review.xlsx

Can you book a meeting with me to go through the entries, we can then make a consolidated list of outstanding questions or queries.

**Comment by pcrjoana** [comment]
@neil-ices-dk and I updated the document to the best of our abilities. I have some follow up to do, and once that's done I will prepare a power BI dashboard. The goal with the dashboard will be to have it linked to a sharepoint list once DIG moves to the new sharepoint. At that point we won't need the document anymore.

The updated list: https://community.ices.dk/Committees/DIG/Master%20documents/Governance%20Evaluations/governance-overview.2024-review.xlsx

**Comment by pcrjoana** [comment]
Apologies for losing a bit track of this - I will be preparing a power BI dashboard within the next week.

**Comment by davidcurrie2001** [comment]
@pcrjoana is there a dashboard you can show at our online meeting next week?

**Comment by pcrjoana** [comment]
> @pcrjoana is there a dashboard you can show at our online meeting next week?

We haven't published this as a sharepoint list yet. I think we're waiting to migrate the DIG github. Any thoughts on this, @neil-ices-dk? I can present the EXCEL file though.

**Comment by davidcurrie2001** [comment]
WGDG updated their previous evaluation using the updated self-evaluation tool.
WGRDBESGOV and WGSMART have subsequently completed self-evaluations using the updated governance group self-evaluation tool.
